src/sqlitestorage.cpp

```diff
@@ -198,63 +198,69 @@ bool SqliteStorage::open()
   sqlite3_busy_timeout( d->mDatabase, 1500 );
 
   /* Create Calendars, Components, etc. tables */
-  query = CREATE_VERSION;
-  sqlite3_exec( d->mDatabase );
+#ifdef Q_OS_UNIX
```

Comment:
I don't see much point to this; better to just #error if Q_OS_UNIX is not defined, I'd say.